op-e2e/e2eutils/wait/waits.go

```diff
@@ -46,7 +46,7 @@ func ForReceipt(ctx context.Context, client *ethclient.Client, hash common.Hash,
 	defer ticker.Stop()
 	for {
 		receipt, err := client.TransactionReceipt(ctx, hash)
-		if errors.Is(err, ethereum.NotFound) {
+		if errors.Is(err, ethereum.NotFound) || (err != nil && err.Error() == "transaction indexing is in progress") {
```

Comment:
this is ugly but unfortunately the underlying error isn't exposed in geth.. I suppose we could expose it in op-geth though.

Reply:
good call, done.